packages/connect-query/src/unary-hooks.ts

```diff
@@ -276,6 +277,8 @@ export const unaryHooks = <I extends Message<I>, O extends Message<O>>({
             input !== disableQuery,
             'queryFn does not accept a disabled query',
           );
+          const valueAtPageParam =
+            input[pageParamKey as unknown as keyof PartialMessage<I>];
```

Comment:
this need to cast suggests to me that the original type was just wrong.  after all: does it really matter if we do ParitalMessage instead of PlainMessage for pageParamKey itself?  Maybe this is more correct this way 0fc87fb6110680afa55b5b36990adb75b75dd0da.  Take a look (the intention is that if we don't like it for any reason we can drop the commit).

Reply:
Oh man, no that looks great. I struggled a little bit with that and just wanted to make sure it works before returning to it. Makes total sense.

Reply:
oh cool, sweet.  I wasn't sure but this also seems good to me too.  let's keep the commit then until someone tells us otherwise (in the future, I mean, beyond this PR).